### PR TITLE
Add docs about Backend Module creation without Extbase

### PR DESCRIPTION
**Pull request by ochorocho**
Shows an alternative way of Backend templating without Extbase which might be a better choice nowadays.

### CODE REVIEW

File: Documentation/ApiOverview/BackendModules/TemplateClass/BackendTemplateViewWithoutExtbase.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
+.. include:: /Includes.rst.txt
+.. index:: ! JOCHEN
```
Comment (susannemoog) [inline Documentation/ApiOverview/BackendModules/TemplateClass/BackendTemplateViewWithoutExtbase.rst:2]: why is Jochen here?

File: Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst [hunk 2]
```diff
@@ -15,6 +15,12 @@ The Backend Template View
    this manual if you need a reference to the old way of programming
    backend modules.
 
+.. tip::
+
+   If you don't want to do extensive data modelling consider working without
+   :ref:`Extbase templating. <backend-modules-template-without-extbase>`
+
+
 Modern backend modules are written using the Extbase/Fluid combination.
```
Comment (susannemoog) [inline Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst:24]: We should probably adjust this sentence to be less "absolute" (maybe "can be written")

File: Documentation/ApiOverview/BackendModules/TemplateClass/BackendTemplateViewWithoutExtbase.rst (hunk 1 above)
Comment (ochorocho) [reply Documentation/ApiOverview/BackendModules/TemplateClass/BackendTemplateViewWithoutExtbase.rst]: 🤦 Sorry, removed it. Was left over code from my tests.

File: Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst (hunk 2 above)
Comment (ochorocho) [reply Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst:26]: Tried to be less "absolute"

Thanks for reviewing

**Review by susannemoog — APPROVED** [review]

**Pull request merged**